diy/diy-straps.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: page
+nav_order: 5
+parent: DIY trackers guide
+---
+# DIY Straps Guide
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Disclaimer
+{:.no_toc}
+*This page is not a definitive/step by step guide to straps.*
+*This page is meant to be used as inspiration to make your own DIY straps.*
+*All examples on this page are community made/provided.*
+
+## Tom Yum's
+
+Tom Yum wanted to prevent straps from sliding. He opted to to directly sew the velcro and use it as the main mounting system, he also chose to mount the trackers using velcro instead of loops to eliminate skin contact with all the parts other the the elastic bands.

```

Comment:
```suggestion
Tom Yum wanted to prevent straps from sliding. He opted to directly sew the velcro and use it as the main mounting system, he also chose to mount the trackers using velcro instead of loops to eliminate skin contact with all the parts other than the elastic bands.
```